config/201-controller-role.yaml

```diff
@@ -62,12 +62,15 @@ metadata:
     app.kubernetes.io/instance: default
     app.kubernetes.io/part-of: pipelines-as-code
 rules:
+  - apiGroups: [""]
+    resources: ["namespaces"]
+    verbs: ["create"]
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["get", "create"]
   - apiGroups: ["pipelinesascode.tekton.dev"]
     resources: ["repositories"]
-    verbs: ["list"]
+    verbs: ["create", "list"]
```

Comment:
This is a bit annoying thing to give! I wish we could separate those capaibilities depending of feature being enabled or not...

Reply:
is it complicated with operator or when doing it with PAC, 

I wonder if we should just disable it by default, document it and let the operator handles it?

I don't think everyone would be using this feature....

Reply:
with operator it will be easy actually depending on what user configure we an update the role

Reply:
https://github.com/openshift-pipelines/pipelines-as-code/issues/933